### DISCUSSION TITLE
How to limit the number of references in chat

### DISCUSSION THREAD

**Discussion by lone17** [khoj-ai/khoj#862] [Q&A] [answered]
Hi, how can I limit the number of retrieved references when using chat ?

**Comment by debanjum** [comment, marked as answer]
You can add file filters [to a conversation](https://docs.khoj.dev/features/chat/#conversation-file-filters) (currently only on web app) or to [any chat message](https://docs.khoj.dev/miscellaneous/advanced/#query-filters) (e.g using `file:"/path/to/your/file.txt"`). 

Does that help limit the references you want Khoj to use? What client are you using Khoj from?
